modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaGetCategoryIdMethodCheck.testjava

```diff
@@ -0,0 +1,11 @@
+package com.liferay.source.formatter.dependencies.upgrade;
+
+public class UpgradeJavaGetCategoryIdMethodCheck {
+
+	public void method(AssetCategory<?> assetCategory) {
+		UnsafeFunction<AssetCategory, Long, Throwable> getCategoryId = assetCategory.getCategoryId;
```

Comment:
@MicaelleSilvaa, the first case was correct before. Sorry about that, the change was about the `AssetCategory::getCategoryId;` case. This syntax does not return a String but instead returns a function. Please adjust the `assetCategory.getCategoryId;` to the way that was before.

Reply:
I will do it